src/Microsoft.DotNet.Darc/src/Darc/Operations/UpdateReviewersOperation.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Darc.Helpers;
+using Microsoft.DotNet.Darc.Options;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Darc.Operations
+{
+    internal class UpdateReviewersOperation : Operation
+    {
+        UpdateReviewersCommandLineOptions _options;
+        public UpdateReviewersOperation(UpdateReviewersCommandLineOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        public override async Task<int> ExecuteAsync()
+        {
+            var subscriptionId = _options.SubscriptionId.Trim();
+            var reviewers = _options.Reviewers.Trim();
+
+            if (string.IsNullOrWhiteSpace(subscriptionId))
+            {
+                Logger.LogError("Please specify a subscription id");
+                return Constants.ErrorCode;
+            }
+
+            // stuff the reviewers into a List
+            var reviewersList = _options.Reviewers.Split(',').ToList();
```

Comment:
You can change the option itself to have a separator and then it returns an IEnumerable

https://github.com/dotnet/arcade-services/blob/master/src/Microsoft.DotNet.Darc/src/Darc/Options/AddSubscriptionCommandLineOptions.cs#L33